cogs/mod.py

```diff
@@ -309,10 +310,24 @@ def check_if_target_is_staff(self, target):
         log_channel = self.bot.get_channel(config.modlog_channel)
         if not channel:
             channel = ctx.channel
-        await channel.purge(limit=limit)
-        msg = f"🗑 **Purged**: {ctx.author.mention} purged {limit} "\
+        purge = await channel.purge(limit=limit)
+        newLimit = len(purge)
+        msg = f"🗑 **Purged**: {ctx.author.mention} purged {newLimit} "\ # More accurate to check for the length of the purged list.
```

Comment:
Line is longer than 80 chars, put comment to a different line if it's necessary, drop it otherwise.

Reply:
Explanation needed.

Reply:
The line with comment has more than 80 characters on it in total, which is too long as per the PEP8 interpretation I follow. The simplest solution is to either remove the comment (it seems unneeded) or move it to a line above this.

Reply:
Sure, I'll remove it in the changes